src/main/kotlin/org/wycliffeassociates/otter/jvm/resourcecardpage/widgets/HighlightableButton.kt

```diff
@@ -0,0 +1,42 @@
+package org.wycliffeassociates.otter.jvm.resourcecardpage.widgets
+
+import com.jfoenix.controls.JFXButton
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.scene.Node
+import org.wycliffeassociates.otter.jvm.resourcecardpage.styles.HighlightableButtonStyles
+import tornadofx.FX.Companion.messages
+import tornadofx.*
+
+abstract class HighlightableButton(btnText: String, btnGraphic: Node) : JFXButton() {
```

Comment:
why is the graphic a node and not a graphic?
why are the text and graphic fields required?
why is the API for this fundamentally different from a button?

Reply:
I suppose the text and graphic fields could just be set in the child classes.
Not sure about the 3rd question